blocks/settlement_test.go

```diff
@@ -36,7 +36,10 @@ func ExampleBlock_WhenChildSettles() {
 
 	// Returns the (possibly empty) slice of blocks that would be settled by the
 	// block being built.
-	_ = parent.WhenChildSettles(settle)
+	_ = Range(parent.LastSettled(), settle)
+	// Returns the (possibly empty) slice of blocks that would be left unsettled
+	// by the block being built.
+	_ = Range(settle, parent)
```

Comment:
Nice :)

I was thinking through the arguments after we chatted and realised it would fall out really elegantly, especially when also looking inside `Block.Settles()`.